arches/app/templates/views/components/search/search-export.htm

```diff
@@ -54,25 +103,35 @@ <h4 class="">
             </div>
         </div>
 
+        <!-- 2. Coordinate Precision -->
         <div class="instruction">
-                <h2 class="">
+                <div class="search-export-instruction-h2 h2">
                     <span data-bind="text: $root.translations.listItemTwoCoordinatePrecision"></span>
-                </h2>
-                <h4 class="">
+                </div>
+                <div class="search-export-instruction-h4 h4">
                     <span data-bind="text: $root.translations.listItemTwoCoordinatePrecisionText"></span>
-                </h4>
+                </div>
         </div>
         <div class="parameter">
-            <input type="number" class="form-control input-md widget-input precision" data-bind="textInput: precision"></input>
+            <input type="number" class="form-control input-md widget-input precision" 
+                data-bind="
+                    textInput: precision,
+                    attr: {
+                        'aria-label': $root.translations.listItemTwoCoordinatePrecision, 
+                        'aria-description': $root.translations.listItemTwoCoordinatePrecisionText
+                    },
+                "
+            ></input>
         </div>
         
+        <!-- 3. Report link -->
         <div class="instruction">
-            <h2 class="">
-                <span data-bind="text: $root.translations.listItemThreeReportLink"></span>
-            </h2>
-            <h4 class="">
-                <span data-bind="text: $root.translations.listItemThreeReportLinkText"></span>
-            </h4>
+            <div class="search-export-instruction-h2 h2">
+                <span data-bind="text: $root.translations.listItemThreeReportLink" id="report-link-header"></span>
+            </div>
+            <!-- <div class="search-export-instruction-h4 h4">
```

Comment:
Moving this text to the subtitle of the switch doesn't look as nice, but it's the easiest way I thought to make it accessible.